### ISSUE TITLE
Error installing sonarr package

### ISSUE THREAD

**Issue by ouija** [OneCDOnly/sherpa#296]
Hey, 

Attempting the switch from **SickGear** to **Sonarr** today, but ran into issue running the `sherpa install sonarr` command, with the following output:
```
sherpa v240912-stable
done: actions complete.

• Package actions started @ 3:53:17 PM, ended @ 3:55:51 PM, elapsed = 2m:34s

• This package action was skipped (and why):
      download OSonarr QPKG (existing file 'OSonarr_240912_x86_64.qpkg' checksum is correct)

• This package action failed (and why):
      install OSonarr QPKG in 2m:32s (For more information: /etc/init.d/osonarr.sh log)
```
And when looking at the log via the `/etc/init.d/osonarr.sh log` command, the error appears to do with `sqlite3`:
```
    677 dbug: (LL) [Info] NzbDrone.Core.Datastore.Migration.Framework.NzbDroneSQLiteProcessor: Performing DB Operation
    678 dbug: (LL) [Error] FluentMigrator.Runner.MigrationRunner: Unable to find an entry point named 'sqlite3_config' in shared library 'sqlite3'.
    679 dbug: (LL)
    680 dbug: (LL) [v4.0.9.2244] System.EntryPointNotFoundException: Unable to find an entry point named 'sqlite3_config' in shared library 'sqlite3'.
```
and further down in the log:
```
    768 dbug: (LL) [Fatal] ConsoleApp: EPIC FAIL!
    769 dbug: (LL)
    770 dbug: (LL) [v4.0.9.2244] NzbDrone.Common.Exceptions.SonarrStartupException: Sonarr failed to start: Error creating main database
    771 dbug: (LL)  ---> System.EntryPointNotFoundException: Unable to find an entry point named 'sqlite3_config' in shared library 'sqlite3'.
```
I attempted to run the `sherpa check` commands but issue still persists -- any suggestions?

EDIT:  Was reaching closed issues  similar to this for more info, and if it's helpful, I did a search for `sqllite3` packages on my system _(as per #275)_ via the `find / -name sqlite3` command, which returns:
```
/share/CACHEDEV3_DATA/.qpkg/qusbcam2/bin/sqlite3
/share/CACHEDEV3_DATA/.qpkg/Entware/bin/sqlite3
/share/CACHEDEV3_DATA/.qpkg/Entware/lib/python3.11/sqlite3
/share/CACHEDEV3_DATA/.qpkg/container-station/usr/bin/sqlite3
/share/CACHEDEV3_DATA/.qpkg/container-station/usr/bin/.libs/sqlite3
/share/CACHEDEV3_DATA/.qpkg/container-station/usr/local/container-station/python/lib/python2.7/sqlite3
/share/CACHEDEV3_DATA/.qpkg/container-station/usr/local/container-station/python3/lib/python3.6/sqlite3
/share/CACHEDEV3_DATA/.qpkg/container-station/system-docker/overlay2/decdbb68c259fda35f17286bbbbe826c9fad4532e186e9333c981ec46fdeab65/diff/usr/bin/sqlite3
/share/CACHEDEV3_DATA/.qpkg/container-station/system-docker/overlay2/decdbb68c259fda35f17286bbbbe826c9fad4532e186e9333c981ec46fdeab65/diff/usr/lib/python2.7/sqlite3
/share/CACHEDEV3_DATA/.qpkg/container-station/system-docker/overlay2/decdbb68c259fda35f17286bbbbe826c9fad4532e186e9333c981ec46fdeab65/diff/usr/lib/python3.5/sqlite3
/share/CACHEDEV3_DATA/.qpkg/container-station/system-docker/overlay2/e0fddc5965b49d270ff0de7efa38d02ac9c6220252624ef3db71893966a6790f/merged/usr/bin/sqlite3
/share/CACHEDEV3_DATA/.qpkg/container-station/system-docker/overlay2/e0fddc5965b49d270ff0de7efa38d02ac9c6220252624ef3db71893966a6790f/merged/usr/lib/python2.7/sqlite3
/share/CACHEDEV3_DATA/.qpkg/container-station/system-docker/overlay2/e0fddc5965b49d270ff0de7efa38d02ac9c6220252624ef3db71893966a6790f/merged/usr/lib/python3.5/sqlite3
/share/CACHEDEV3_DATA/.qpkg/container-station/system-docker/overlay2/a2b6f028abdef1b8a1f7b09305e9d984b4dce405905a3cdc8c1bf0e716efdde4/diff/usr/lib/python2.7/sqlite3
/share/CACHEDEV3_DATA/Container/container-station-data/image/lxc/ubuntu-bionic/image/rootfs/usr/lib/python3.6/sqlite3
/share/CACHEDEV3_DATA/Container/container-station-data/lib/lxc/qnap-container/rootfs/usr/lib/python3.8/sqlite3
/share/CACHEDEV3_DATA/Container/container-station-data/lib/docker/overlay2/107ea85aeb2746e13b8f9a514d5b6d39f8a13505c20760785efc807c6e90edd5/diff/usr/lib/python3.9/sqlite3
/share/CACHEDEV3_DATA/Container/container-station-data/lib/docker/overlay2/8ab90ff0b787a109c29468307300f95b22de7d6b085d25b8b3a49ab10f4d1386/merged/usr/lib/python3.9/sqlite3
/share/CACHEDEV3_DATA/Container/container-station-data/lib/docker/overlay2/23024bc98bdf4eb4154b59eae77e7747ca48fbb7035db56ca6ba3d8e6af5e8e5/diff/usr/lib/python2.7/sqlite3
/share/CACHEDEV3_DATA/Container/container-station-data/lib/docker/overlay2/8d9a6c2a5359ff8b67657217b8b3840281726fcd40bcec60ab907f573a4db3b2/diff/usr/lib/python3.5/sqlite3
```
Not sure if helpful but figured I'd provide this.

Note that I currently have the `Entware`, `Par2`, `Par2turbo`, `SABnzbd`, `OSickGear` and `Unrar` packages instead via **sherpa** _(along with this failed `OSonarr` package)_ and also have `ContainerStation` installed, along with this `QUSBCam2` qpkg from the QNAP store.

**Comment by ouija** [comment]
I wanted to avoid it since I have additional packages installed via Entware _(but this may also be the cause of the failure)_ so I am attempting a `sherpa reinstall ew` command now and will report back...

EDIT:  No difference, same error in the logs when attempting re-install of sonarr package.

**Comment by OneCDOnly** [comment]
Hi @ouija and thanks for reporting this problem. I saw it just as I was leaving for work this morning. 😄 

What is your NAS model and firmware version please? 

Thanks for running those checks. The sqlite3 files we were looking-for in that linked issue are sqlite3 binaries that can be used for "signing" QPKGs when Entware hasn't been installed yet.  

You already have Entware installed, so sherpa will use the Entware sqlite3 package for signing.

I wonder if one of the other sqlite3 binaries is causing a problem with Sonarr though? I'll do some testing locally and report back. 

In the meantime, can you please clean your Sonarr installation?
```
/etc/init.d/osonarr.sh clean debug
```
If it's not started, you'll need to start it manually:
```
/etc/init.d/osonarr.sh start debug
```

**Comment by OneCDOnly** [comment]
Also, you shouldn't need both Par2 and Par2turbo QPKGs installed. Only one is required by SABnzbd for repairing downloads. If you uninstall them both, then run a `sherpa check`, the correct one will be installed again.

**Comment by OneCDOnly** [comment]
No problems at this end. I've been unable to replicate the issue. :(

It might have been just a faulty Sonarr download (it-happens). Let's see if a `clean` fixes it. 

On a side-note: I did find a couple of issues with the `sqlite3` detection logic which I've now fixed. 👍🏽

**Comment by ouija** [comment]
I have the **QNAP TS-870 Pro**.

Thanks for the tip about the two Par2 packages :) Removed them both and `sherpa check` installed Par2turbo by default.

Still appear to be having issues running Sonarr however;  Running `/etc/init.d/osonarr.sh clean debug` has it timing out on `test for port` due to same sqlite3 error:

```
   1532 > test for port '8989' response (no-more than 120 seconds): failed
   1533 no response on configured port(s)
   1534 port tests failed!
   1535 dbug: (LL) adding external log to main log ...
   1536 dbug: (LL) ----------------------------------------------------------------------------------------------------
   1537 dbug: (LL) = log file: '/var/log/OSonarr.screen.log'
   1538 dbug: (LL) command: 'export DOTNET_SYSTEM_GLOBALIZATION_INVARIANT=1 TMPDIR=/share/CACHEDEV3_DATA/.qpkg/OSonarr/tmp;/share/CACHEDEV3_DATA/.qpkg/OSonarr/repo-cache/Son
   1538 arr/Sonarr --nobrowser --data=/share/CACHEDEV3_DATA/.qpkg/OSonarr/config'
   1539 dbug: (LL) [Info] Bootstrap: Starting Sonarr - /share/CACHEDEV3_DATA/.qpkg/OSonarr/repo-cache/Sonarr/Sonarr - Version 4.0.9.2244
   1540 dbug: (LL) [Info] AppFolderInfo: Data directory is being overridden to [/share/CACHEDEV3_DATA/.qpkg/OSonarr/config]
   1541 dbug: (LL) [Debug] Bootstrap: Console selected
   1542 dbug: (LL) [Info] AppFolderInfo: Data directory is being overridden to [/share/CACHEDEV3_DATA/.qpkg/OSonarr/config]
   1543 dbug: (LL) [Info] AppFolderInfo: Data directory is being overridden to [/share/CACHEDEV3_DATA/.qpkg/OSonarr/config]
   1544 dbug: (LL) [Info] MigrationController: *** Migrating data source=/share/CACHEDEV3_DATA/.qpkg/OSonarr/config/sonarr.db;cache size=-20000;datetimekind=Utc;journal mode
   1544 =Wal;pooling=True;version=3;busytimeout=100 ***
   1545 dbug: (LL) [Info] FluentMigrator.Runner.MigrationRunner: DatabaseEngineVersionCheck migrating
   1546 dbug: (LL) [Info] FluentMigrator.Runner.MigrationRunner: PerformDBOperation
   1547 dbug: (LL) [Info] NzbDrone.Core.Datastore.Migration.Framework.NzbDroneSQLiteProcessor: Performing DB Operation
   1548 dbug: (LL) [Error] FluentMigrator.Runner.MigrationRunner: Unable to find an entry point named 'sqlite3_config' in shared library 'sqlite3'.
   1549 dbug: (LL)
   1550 dbug: (LL) [v4.0.9.2244] System.EntryPointNotFoundException: Unable to find an entry point named 'sqlite3_config' in shared library 'sqlite3'.
   1551 dbug: (LL)    at System.Data.SQLite.UnsafeNativeMethods.sqlite3_config_none(SQLiteConfigOpsEnum op)
   1552 dbug: (LL)    at System.Data.SQLite.SQLite3.StaticIsInitialized()
   1553 dbug: (LL)    at System.Data.SQLite.SQLiteLog.PrivateInitialize(String className)
   1554 dbug: (LL)    at System.Data.SQLite.SQLiteLog.Initialize(String className)
   1555 dbug: (LL)    at System.Data.SQLite.SQLiteConnection..ctor(String connectionString, Boolean parseViaFramework)
   1556 dbug: (LL)    at System.Data.SQLite.SQLiteConnection..ctor(String connectionString)
   1557 dbug: (LL)    at System.Data.SQLite.SQLiteConnection..ctor()
   1558 dbug: (LL)    at System.Data.SQLite.SQLiteFactory.CreateConnection()
   1559 dbug: (LL)    at FluentMigrator.Runner.Processors.GenericProcessorBase.<>c__DisplayClass6_1.<.ctor>b__1()
   1560 dbug: (LL)    at System.Lazy`1.ViaFactory(LazyThreadSafetyMode mode)
   1561 dbug: (LL)    at System.Lazy`1.ExecutionAndPublication(LazyHelper executionAndPublication, Boolean useDefaultConstructor)
   1562 dbug: (LL)    at System.Lazy`1.CreateValue()
   1563 dbug: (LL)    at System.Lazy`1.get_Value()
   1564 dbug: (LL)    at FluentMigrator.Runner.Processors.GenericProcessorBase.get_Connection()
   1565 dbug: (LL)    at FluentMigrator.Runner.Processors.GenericProcessorBase.EnsureConnectionIsOpen()
   1566 dbug: (LL)    at FluentMigrator.Runner.Processors.SQLite.SQLiteProcessor.Process(PerformDBOperationExpression expression)
   1567 dbug: (LL)    at FluentMigrator.Expressions.PerformDBOperationExpression.ExecuteWith(IMigrationProcessor processor)
   1568 dbug: (LL)    at FluentMigrator.Runner.MigrationRunner.<>c__DisplayClass80_0.<ExecuteExpressions>b__1()
   1569 dbug: (LL)    at FluentMigrator.Runner.StopWatch.Time(Action action)
   1570 dbug: (LL)    at FluentMigrator.Runner.MigrationRunner.ExecuteExpressions(ICollection`1 expressions)
   1571 dbug: (LL)
   1572 dbug: (LL) [Fatal] Microsoft.AspNetCore.Hosting.Diagnostics: Application startup exception
   1573 dbug: (LL)
   1574 dbug: (LL) [v4.0.9.2244] NzbDrone.Common.Exceptions.SonarrStartupException: Sonarr failed to start: Error creating main database
      1575 dbug: (LL)  ---> System.EntryPointNotFoundException: Unable to find an entry point named 'sqlite3_config' in shared library 'sqlite3'.
   1576 dbug: (LL)    at System.Data.SQLite.UnsafeNativeMethods.sqlite3_config_none(SQLiteConfigOpsEnum op)
   1577 dbug: (LL)    at System.Data.SQLite.SQLite3.StaticIsInitialized()
   1578 dbug: (LL)    at System.Data.SQLite.SQLiteLog.PrivateInitialize(String className)
   1579 dbug: (LL)    at System.Data.SQLite.SQLiteLog.Initialize(String className)
   1580 dbug: (LL)    at System.Data.SQLite.SQLiteConnection..ctor(String connectionString, Boolean parseViaFramework)
   1581 dbug: (LL)    at System.Data.SQLite.SQLiteConnection..ctor(String connectionString)
   1582 dbug: (LL)    at System.Data.SQLite.SQLiteConnection..ctor()
   1583 dbug: (LL)    at System.Data.SQLite.SQLiteFactory.CreateConnection()
   1584 dbug: (LL)    at FluentMigrator.Runner.Processors.GenericProcessorBase.<>c__DisplayClass6_1.<.ctor>b__1()
   1585 dbug: (LL)    at System.Lazy`1.ViaFactory(LazyThreadSafetyMode mode)
   1586 dbug: (LL)    at System.Lazy`1.ExecutionAndPublication(LazyHelper executionAndPublication, Boolean useDefaultConstructor)
   1587 dbug: (LL)    at System.Lazy`1.CreateValue()
   1588 dbug: (LL)    at System.Lazy`1.get_Value()
   1589 dbug: (LL)    at FluentMigrator.Runner.Processors.GenericProcessorBase.get_Connection()
   1590 dbug: (LL)    at FluentMigrator.Runner.Processors.GenericProcessorBase.EnsureConnectionIsOpen()
   1591 dbug: (LL)    at FluentMigrator.Runner.Processors.SQLite.SQLiteProcessor.Process(PerformDBOperationExpression expression)
   1592 dbug: (LL)    at FluentMigrator.Expressions.PerformDBOperationExpression.ExecuteWith(IMigrationProcessor processor)
   1593 dbug: (LL)    at FluentMigrator.Runner.MigrationRunner.<>c__DisplayClass80_0.<ExecuteExpressions>b__1()
   1594 dbug: (LL)    at FluentMigrator.Runner.StopWatch.Time(Action action)
   1595 dbug: (LL)    at FluentMigrator.Runner.MigrationRunner.ExecuteExpressions(ICollection`1 expressions)
   1596 dbug: (LL)    at FluentMigrator.Runner.MigrationRunner.ExecuteMigration(IMigration migration, Action`2 getExpressions)
   1597 dbug: (LL)    at FluentMigrator.Runner.MigrationRunner.ApplyMigrationUp(IMigrationInfo migrationInfo, Boolean useTransaction)
   1598 dbug: (LL)    at FluentMigrator.Runner.MigrationRunner.ApplyMaintenance(MigrationStage stage, Boolean useAutomaticTransactionManagement)
   1599 dbug: (LL)    at FluentMigrator.Runner.MigrationRunner.MigrateUp(Int64 targetVersion, Boolean useAutomaticTransactionManagement)
   1600 dbug: (LL)    at FluentMigrator.Runner.MigrationRunner.MigrateUp(Boolean useAutomaticTransactionManagement)
   1601 dbug: (LL)    at FluentMigrator.Runner.MigrationRunner.MigrateUp()
   1602 dbug: (LL)    at NzbDrone.Core.Datastore.Migration.Framework.MigrationController.Migrate(String connectionString, MigrationContext migrationContext, DatabaseType dat
   1602 abaseType) in ./Sonarr.Core/Datastore/Migration/Framework/MigrationController.cs:line 80
   1603 dbug: (LL)    at NzbDrone.Core.Datastore.DbFactory.CreateMain(String connectionString, MigrationContext migrationContext, DatabaseType databaseType) in ./Sonarr.Core
   1603 /Datastore/DbFactory.cs:line 116
   1604 dbug: (LL)    --- End of inner exception stack trace ---
   1605 dbug: (LL)    at NzbDrone.Core.Datastore.DbFactory.CreateMain(String connectionString, MigrationContext migrationContext, DatabaseType databaseType) in ./Sonarr.Core
   1605 /Datastore/DbFactory.cs:line 164
   1606 dbug: (LL)    at NzbDrone.Core.Datastore.DbFactory.Create(MigrationContext migrationContext) in ./Sonarr.Core/Datastore/DbFactory.cs:line 71
   1607 dbug: (LL)    at NzbDrone.Core.Datastore.DbFactory.Create(MigrationType migrationType) in ./Sonarr.Core/Datastore/DbFactory.cs:line 59
   1608 dbug: (LL)    at NzbDrone.Core.Datastore.Extensions.CompositionExtensions.<>c.<AddDatabase>b__0_0(IDbFactory f) in ./Sonarr.Core/Datastore/Extensions/CompositionExte
   1608 nsions.cs:line 10
   1609 dbug: (LL)    at DryIoc.Registrator.ToFuncWithObjParams[D1,TService](Func`2 f, Object d1) in /_/src/DryIoc/Container.cs:line 8067
   1610 dbug: (LL)    at DryIoc.Interpreter.TryInterpretFuncInvoke(IResolverContext r, MethodCallExpression e, IParameterProvider paramExprs, Object paramValues, ParentLambd
   1610 aArgs parentArgs, Object& result) in /_/src/DryIoc/Container.cs:line 3727
   1611 dbug: (LL)    at DryIoc.Interpreter.TryInterpretMethodCall(IResolverContext r, MethodCallExpression callExpr, IParameterProvider paramExprs, Object paramValues, Pare
   1611 ntLambdaArgs parentArgs, Object& result) in /_/src/DryIoc/Container.cs:line 3525
   1612 dbug: (LL)    at DryIoc.Interpreter.TryInterpret(IResolverContext r, Expression expr, IParameterProvider paramExprs, Object paramValues, ParentLambdaArgs parentArgs,
   1612  Object& result) in /_/src/DryIoc/Container.cs:line 3176
   1613 dbug: (LL)    at DryIoc.Interpreter.TryInterpretSingletonAndUnwrapContainerException(IResolverContext r, Expression expr, ImMapEntry`1 itemRef, Object& result) in /_
   1613 /src/DryIoc/Container.cs:line 3105
   1614 dbug: (LL)    at DryIoc.Factory.ApplyReuse(Expression serviceExpr, Request request) in /_/src/DryIoc/Container.cs:line 11136
   1615 dbug: (LL)    at DryIoc.Factory.GetExpressionOrDefault(Request request) in /_/src/DryIoc/Container.cs:line 11055
   1616 dbug: (LL)    at DryIoc.Container.ResolveAndCacheKeyed(Int32 serviceTypeHash, Type serviceType, Object serviceKey, IfUnresolved ifUnresolved, Object scopeName, Type
   1616 requiredServiceType, Request preResolveParent, Object[] args) in /_/src/DryIoc/Container.cs:line 541
   1617 dbug: (LL)    at DryIoc.Container.DryIoc.IResolver.Resolve(Type serviceType, Object serviceKey, IfUnresolved ifUnresolved, Type requiredServiceType, Request preResol
   1617 veParent, Object[] args) in /_/src/DryIoc/Container.cs:line 469
   1618 dbug: (LL)    at DryIoc.Interpreter.TryInterpretMethodCall(IResolverContext r, MethodCallExpression callExpr, IParameterProvider paramExprs, Object paramValues, Pare
   1618 ntLambdaArgs parentArgs, Object& result) in /_/src/DryIoc/Container.cs:line 3661
   1619 dbug: (LL)    at DryIoc.Interpreter.TryInterpret(IResolverContext r, Expression expr, IParameterProvider paramExprs, Object paramValues, ParentLambdaArgs parentArgs,
   1619  Object& result) in /_/src/DryIoc/Container.cs:line 3188
   1620 dbug: (LL)    at DryIoc.Interpreter.TryInterpretNestedLambdaBodyAndUnwrapException(IResolverContext r, Expression bodyExpr, IParameterProvider paramExprs, Object par
   1620 amValues, ParentLambdaArgs parentArgs) in /_/src/DryIoc/Container.cs:line 3471
   1621 dbug: (LL)    at DryIoc.Interpreter.<>c__DisplayClass5_0.<TryInterpretNestedLambda>b__0() in /_/src/DryIoc/Container.cs:line 3370
   1622 dbug: (LL)    at DryIoc.Interpreter.<>c__DisplayClass9_0`1.<ConvertFunc>b__0() in /_/src/DryIoc/Container.cs:line 3481
   1623 dbug: (LL)    at System.Lazy`1.ViaFactory(LazyThreadSafetyMode mode)
   1624 dbug: (LL)    at System.Lazy`1.ExecutionAndPublication(LazyHelper executionAndPublication, Boolean useDefaultConstructor)
   1625 dbug: (LL)    at System.Lazy`1.CreateValue()
   1626 dbug: (LL)    at System.Lazy`1.get_Value()
   1627 dbug: (LL)    at NzbDrone.Host.Startup.Configure(IApplicationBuilder app, IContainer container, IStartupContext startupContext, Lazy`1 mainDatabaseFactory, Lazy`1 lo
   1627 gDatabaseFactory, DatabaseTarget dbTarget, ISingleInstancePolicy singleInstancePolicy, InitializeLogger initializeLogger, ReconfigureLogging reconfigureLogging, IApp
   1627 FolderFactory appFolderFactory, IProvidePidFile pidFileProvider, IConfigFileProvider configFileProvider, IRuntimeInfo runtimeInfo, IFirewallAdapter firewallAdapter,
   1627 IEventAggregator eventAggregator, SonarrErrorPipeline errorHandler) in ./Sonarr.Host/Startup.cs:line 225
   1628 dbug: (LL)    at System.RuntimeMethodHandle.InvokeMethod(Object target, Span`1& arguments, Signature sig, Boolean constructor, Boolean wrapExceptions)
   1629 dbug: (LL)    at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
   1630 dbug: (LL)    at Microsoft.AspNetCore.Hosting.ConfigureBuilder.Invoke(Object instance, IApplicationBuilder builder)
   1631 dbug: (LL)    at Microsoft.AspNetCore.Hosting.ConfigureBuilder.<>c__DisplayClass4_0.<Build>b__0(IApplicationBuilder builder)
   1632 dbug: (LL)    at Microsoft.AspNetCore.Hosting.GenericWebHostBuilder.<>c__DisplayClass15_0.<UseStartup>b__1(IApplicationBuilder app)
   1633 dbug: (LL)    at Microsoft.AspNetCore.Mvc.Filters.MiddlewareFilterBuilderStartupFilter.<>c__DisplayClass0_0.<Configure>g__MiddlewareFilterBuilder|0(IApplicationBuild
   1633 er builder)
   1634 dbug: (LL)    at Microsoft.AspNetCore.Hosting.GenericWebHostService.StartAsync(CancellationToken cancellationToken)
   1635 dbug: (LL)
   1636 dbug: (LL)
   1637 dbug: (LL)
   1638 dbug: (LL) [Fatal] ConsoleApp: EPIC FAIL!
   1639 dbug: (LL)
   1640 dbug: (LL) [v4.0.9.2244] NzbDrone.Common.Exceptions.SonarrStartupException: Sonarr failed to start: Error creating main database
   1641 dbug: (LL)  ---> System.EntryPointNotFoundException: Unable to find an entry point named 'sqlite3_config' in shared library 'sqlite3'.
   1642 dbug: (LL)    at System.Data.SQLite.UnsafeNativeMethods.sqlite3_config_none(SQLiteConfigOpsEnum op)
   1643 dbug: (LL)    at System.Data.SQLite.SQLite3.StaticIsInitialized()
   1644 dbug: (LL)    at System.Data.SQLite.SQLiteLog.PrivateInitialize(String className)
   1645 dbug: (LL)    at System.Data.SQLite.SQLiteLog.Initialize(String className).............
```
Just for info, a `which sqlite3` shows:
`/opt/bin/sqlite3`

Also, don't know if related, but a couple years back I did have Sonarr running on this machine using a different `QSonarr` package _(since removed)_ but figured I'd mention it..

**Comment by ouija** [comment]
I'm testing out the [linuxserver/sonarr](https://hub.docker.com/r/linuxserver/sonarr) docker package _(through container station)_ now and may end up going this route, since it seems to be functional.   But I haven't given up on _sherpa_ yet...

**Comment by OneCDOnly** [comment]
No worries, you might need to switch anyway as I'm not sure what's causing this error. This may be one for the Sonarr developers to investigate. 

I've found mention of this same error (with Emby) that occurred because the libc version in QTS was/is ancient. Can you post your libc version please?
```
ldd --version
```

**Comment by ryanrgi** [comment]
```
ldd (GNU libc) 2.21
```

**Comment by ouija** [comment]
**ldd (GNU libc) 2.21**

**Comment by OneCDOnly** [comment]
Thats interesting... not what I thought it would be. 

I'm running a TS-251+ (x86-64) with QTS 5.1.8 and it shows the same version:
```
[/share/Public] # ldd --version
ldd (GNU libc) 2.21
Copyright (C) 2015 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
Written by Roland McGrath and Ulrich Drepper.
```
But, Sonarr runs without issue. 

I can only suggest if you feel like investigating it further with the developers, maybe they can find the problem. I've no skills troubleshooting .NET applications.

**Comment by ouija** [comment]
No worries -- I've managed to get it running flawlessly now using this [linuxserver/sonarr](https://hub.docker.com/r/linuxserver/sonarr) docker image in container station, and I'm really impressed with it..

So much so that I have also switched over to using the [linuxserver/sabnzbd](https://hub.docker.com/r/linuxserver/sabnzbd) docker container as well for consistency sake, and also fired up [linuxserver/radarr](https://hub.docker.com/r/linuxserver/radaar) now as well.

Still using sherpa for Entware updates, and may use additional packages in the future.

Thanks for authoring such a great tool for QNAP users!   Maybe one day I'll revist the issues the sonarr package has with my TS-870 Pro, but I'm just happy to have a working solution for the time being.

**Comment by ouija** [comment]
No worries -- I've managed to get it running flawlessly now using this [linuxserver/sonarr](https://hub.docker.com/r/linuxserver/sonarr) docker image in container station, and I'm really impressed with it..

So much so that I have also switched over to using the [linuxserver/sabnzbd](https://hub.docker.com/r/linuxserver/sabnzbd) docker container as well for consistency sake, and also fired up [linuxserver/radarr](https://hub.docker.com/r/linuxserver/radarr) now as well.

Still using sherpa for Entware updates, and may use additional packages in the future.

Thanks for authoring such a great tool for QNAP users!   Maybe one day I'll revist the issues the sonarr package has with my TS-870 Pro, but I'm just happy to have a working solution for the time being.
